src/Core/src/ImageSources/Android/Glide/MauiAppGlideModule.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using Android.Content;
+using Android.Graphics;
+using Android.Runtime;
+using Bumptech.Glide;
+using Java.IO;
+
+namespace Microsoft.Maui.BumptechGlide
+{
+	// This class name is very specific as it is directly loaded.
+	[Register("com.bumptech.glide.GeneratedAppGlideModuleImpl")]
+	public class MauiAppGlideModule : GeneratedAppGlideModule
```

Comment:
This is a custom thing for Glide and is generated by a gradle task. But, since we are not using gradle, I manually create it and manually add the modules. This allows us to decode images from a managed stream without copying. However, if this is not desired, then we can decode my first copying to a Java stream and then decoding that.